src/core/core.controller.js

```diff
@@ -1136,6 +1136,15 @@ class Chart {
     return this._plugins.notify(this, hook, args, filter);
   }
 
+  /**
+   * Check if a plugin with the specific ID is registered
```

Comment:
The `init` plugins are also not disabled.

So
```js
options: {
  plugins: {
    filler: false
  }
}
```

should (already) make it return false event if registered.